lib/testing/src/lib/protractor/core/models/user.model.ts

```diff
@@ -33,7 +33,7 @@ export class UserModel {
     id: number;
 
     constructor(details: any = {}) {
-        const EMAIL_DOMAIN = browser.params?.testConfig?.emailDomain ? browser.params.testConfig.emailDomain : 'example.com';
+        const EMAIL_DOMAIN = browser.params?.testConfig?.emailDomain ? browser.params.testConfig.emailDomain : 'default.com';
```

Comment:
I'd rather have this fail or be `example.com` as in https://www.rfc-editor.org/rfc/rfc2606.html default.com belongs to a company

Reply:
done, thanks